pkg/controllers/user/monitoring/metricExpression.go

```diff
@@ -1586,7 +1587,7 @@ metadata:
     component: cluster
     details: "false"
     level: cluster
-    metric: cpu-usage-seconds-sum-rate
+    graph: cpu-usage-seconds-sum-rate
```

Comment:
why we need to change label? Metric label and component label combine to the name. If a graph have multiple metric, you can add graph label to select them all.

Reply:
Rollbacked